packages/tpetra/scripts/PerformanceTesting/crusher/configureBuild.sh

```diff
@@ -34,38 +34,46 @@ rm -Rf CMakeCache.txt CMakeFiles
 
 # See https://docs.olcf.ornl.gov/systems/crusher_quick_start_guide.html#compiling for tips on compiling.
 
+#export HIPFLAGS="--amdgpu-target=gfx90a"
+#PE_MPICH_GTL_DIR_amd_gfx90a="-L${CRAY_MPICH_ROOTDIR}/gtl/lib"
+#PE_MPICH_GTL_LIBS_amd_gfx90a="-lmpi_gtl_hsa"
+
 THE_REAL_NETCDF_C_ROOT="${CRAY_NETCDF_DIR}/crayclang/10.0"
 
+#CMAKE_OPTIONS="--debug-trycompile"
+CMAKE_OPTIONS=""
+
 ARGS=(
+    -GNinja
+    -D Trilinos_TEST_CATEGORIES=PERFORMANCE
+    -D Trilinos_ENABLE_Fortran:BOOL=OFF
     -D CMAKE_INSTALL_PREFIX:PATH="${trilinos_install_dir}"
     -D CMAKE_BUILD_TYPE:STRING=$BUILD_TYPE
 
-    -GNinja
-
-    -D CMAKE_CXX_FLAGS:STRING="-I${ROCM_PATH}/include"
-    -D Trilinos_EXTRA_LINK_FLAGS:STRING="-L${ROCM_PATH}/lib -lamdhip64"
-
-
+    #-D MPI_USE_COMPILER_WRAPPERS:BOOL=ON
     # JHU: 2022-02-07 I've been unable to get Trilinos to build with the Cray MPI wrappers.
     # Instead, compile with hipcc directly.
     -D MPI_USE_COMPILER_WRAPPERS:BOOL=OFF
     -D MPI_EXEC_NUMPROCS_FLAG:STRING="--ntasks"
     -D MPI_EXEC:STRING="srun"
-    -D CMAKE_CXX_COMPILER:PATH="/opt/rocm-4.5.2/bin/hipcc"
+    #-D CMAKE_CXX_FLAGS:STRING="-I${ROCM_PATH}/include"
+    -D Trilinos_EXTRA_LINK_FLAGS:STRING="-L${ROCM_PATH}/lib -lamdhip64"
+    -D CMAKE_CXX_COMPILER:PATH="$HIP_PATH/bin/hipcc"
     -D CMAKE_CXX_FLAGS:STRING="-I${CRAY_MPICH_DIR}/include"
-    -D BUILD_SHARED_LIBS:BOOL=OFF
-    -D Trilinos_EXTRA_LINK_FLAGS:STRING="-L${CRAY_MPICH_DIR}/lib -lmpi -L${CRAY_MPICH_ROOTDIR}/gtl/lib -lmpi_gtl_hsa"
+    -D CMAKE_C_COMPILER:PATH="$HIP_PATH/bin/hipcc"
+    -D CMAKE_C_FLAGS:STRING="-I${CRAY_MPICH_DIR}/include"
+    -D Trilinos_EXTRA_LINK_FLAGS:STRING="-L${CRAY_MPICH_DIR}/lib -lmpi -L${CRAY_MPICH_ROOTDIR}/gtl/lib -lmpi_gtl_hsa -L/${CRAY_BINUTILS_VERSION}/cce/x86_64/lib -lcraymath -lu -lf"
```

Comment:
I am not a big fan of `-L/${CRAY_BINUTILS_VERSION}/cce/x86_64/lib` but for some reason Cray does not provide a variable that points to the toolchain libraries : (